classes.py

```diff
@@ -109,14 +136,29 @@ def adjustScore(self, adj):
 		self.score = self.score + adj
 	
 	def receiveCard(self, card):
+		"""adds the given card to the players deck"""
 		self.deck.addCard(card)
 		return True
 
 	def updateScore(self):
-		self.score = sum([float(card.value) for card in self.deck.cards])
+		"""calculates the player score based on cards in their deck(hand)"""
+		if not self.isdealer:
+			self.score = sum([float(card.value) for card in self.deck.cards])
+		else:
+			print("Dealer cannot have a score")
+			return False
+
+	def resetPlayer(self):
+		"""resets a players score and deck"""
```

Comment:
Where does it reset the score?

Reply:
im not sure why it isnt showing the code below that